pkg/ingester/ingester_test.go

```diff
@@ -10216,7 +10216,7 @@ func TestIngester_PushWithSampledErrors(t *testing.T) {
 	errorSampleRate := 5
 	now := time.Now()
 
-	users := []string{"test", "tset"}
+	users := []string{"user-1", "user-2"}
```

Comment:
lol, why would you change **that**? :)

Reply:
Because I don't like get tricked when I look at failed assertions.